src/main/java/org/apache/accumulo/access/BytesWrapper.java

```diff
@@ -118,18 +111,7 @@ public String toString() {
    * a copy of the input buffer
    */
   void set(byte[] data, int offset, int length) {
-    if (offset < 0) {
-      throw new IllegalArgumentException("Offset cannot be negative. length = " + offset);
-    }
-    if (length < 0) {
-      throw new IllegalArgumentException("Length cannot be negative. length = " + length);
-    }
-    if ((offset + length) > data.length) {
-      throw new IllegalArgumentException(
-          "Sum of offset and length exceeds data length. data.length = " + data.length
-              + ", offset = " + offset + ", length = " + length);
-    }
-
+    Objects.checkFromIndexSize(offset, length, data.length);
```

Comment:
I like that this method is provided built-in. I don't like that it takes 3 ints, and 2 of them have the same semantics of "size of byte string". I had to quadruple check the javadoc and the arguments to make sure the method was being called correctly. I hate that it'd be really easy to mess this up. That said, this looks correct to me.